### ISSUE TITLE
Does fxlauncher works with spring boot

### ISSUE THREAD

**Issue by dhavalsolanki** [edvin/fxlauncher#175]
Hello,

I am implementing fxlauncher in my spring boot project. My main class code is as below

![Capture](https://user-images.githubusercontent.com/5657449/64335412-6089e680-cff8-11e9-91a4-27a71db2f83e.PNG)

It is creating jar file and but when i am running jar file i am getting error as below

![Capture1](https://user-images.githubusercontent.com/5657449/64335607-d1c99980-cff8-11e9-842a-0f1483f26446.PNG)

I just want to know that does that way is supported can i run spring boot code in fxlauncher main class

**Comment by edvin** [comment]
Yes, it does. Looks like you don't have the servlet api on your classpath though. More Spring Boot info here:

https://github.com/edvin/fxlauncher/issues/96
